packages/driver/src/cypress/commands.ts

```diff
@@ -145,6 +215,20 @@ export default {
           })
         }
 
+        if (_.keys(reservedCommandNames).includes(name)) {
```

Comment:
```suggestion
        if (reservedCommandNames[name]) {
```

This way, the lookup is [O(1) instead of O(n)](https://rob-bell.net/2009/06/a-beginners-guide-to-big-o-notation). `Array#includes` has to loop through each item until it finds a match (or loops through every item if there isn't a match), where-as `object[key]` is a constant lookup.